Libraries/Text/TextInput/RCTBaseTextInputView.m

```diff
@@ -26,6 +26,9 @@
 @implementation RCTBaseTextInputView {
   __weak RCTBridge *_bridge;
   __weak id<RCTEventDispatcherProtocol> _eventDispatcher;
+
+  NSInteger _ghostTextPosition; // only valid if _ghostText != nil [macOS]
```

Comment:
We don't ever set this right? It's always at the IP, aka `selection.start`?

Reply:
Also nit move the comment to after [macOS] so it's easier to search for all the macOS tags: `// [macOS] only valid if _ghostText != nil`

Reply:
It should be, but also serves as insurance against selection unexpectedly changing and us having some hope of possibly removing the ghost text anyway.